app/src/main/java/org/mozilla/focus/fragment/FirstrunFragment.kt

```diff
@@ -97,6 +70,34 @@ class FirstrunFragment : Fragment(), View.OnClickListener {
         }
     }
 
+    @Suppress("MagicNumber")
+    private fun setPager() {
```

Comment:
This refactoring while not needed for the ticket is a nice touchup 👍. 
Could the method be named `setupPager`? Just `set` to me says this is a setter.